packages/react-scripts/scripts/start.js

```diff
@@ -258,6 +259,31 @@ function run(port) {
   runDevServer(host, port, protocol);
 }
 
+function isProcessAReactApp(processCommand) {
+  return /^node .*react-scripts\/scripts\/start\.js\s?$/.test(processCommand);
+}
+
+function getProcessForPort(port) {
+  var execOptions = { encoding: 'utf8' };
+
+  try {
+    var processIds = execSync('lsof -i:' + port + ' -P -t', execOptions).match(/(\S+)/g);
+
+    var processCommandsAndDirectories = processIds.map(function(processId) {
+      var processCommand = execSync('ps -o command -p ' + processId + ' | sed -n 2p', execOptions);
+      if (isProcessAReactApp(processCommand)) {
+        processCommand = 'create-react-app\n';
```

Comment:
Can this use the project name from its `package.json` instead of `create-react-app`?